packages/contracts/util/constants.ts

```diff
@@ -2,7 +2,7 @@ export const constants = {
     NULL_BYTES: '0x',
     INVALID_OPCODE: 'invalid opcode',
     REVERT: 'revert',
-    TESTRPC_NETWORK_ID: 50,
+    TESTRPC_NETWORK_ID: 50 as 50,
```

Comment:
Hm, this is going to be really confusing for TS newbies :( Is there no way around this?

Reply:
I'd really like to not do it that way, but I described the reasons in PR description. Maybe the gains here are smaller than losses and we can just abandon that PR?

Reply:
Let's remove this.